packages/shared/lib/participation/participation.ts

```diff
@@ -91,7 +91,19 @@ export const canParticipate = (eventState: ParticipationEventState): boolean =>
  *
  * @returns {boolean}
  */
-export const canAccountParticipate = (account: WalletAccount): boolean => account?.rawIotaBalance >= DUST_THRESHOLD
+export const canAccountParticipate = (account: WalletAccount): AccountParticipationAbility => {
+    if (account?.rawIotaBalance < DUST_THRESHOLD) {
+        return AccountParticipationAbility.NoHasDustAmount
+    } else if (
+        account?.messages.some((message) => {
```

Comment:
The formatting for the second conditional is a bit strange here... might be a result of Prettier's formatting.

Reply:
Updated.